test/run

```diff
@@ -162,6 +163,25 @@ test_config_writeable() {
   docker run --rm "${IMAGE_NAME}" /bin/bash -c "${run_cmd}"
 }
 
+test_home_not_in_webroot() {
+  cid_file=$CID_FILE_DIR/$(mktemp -u -p . --suffix .cid)
+  docker run -d --cidfile=${cid_file} ${IMAGE_NAME}-testapp
+
+  # Wait for container to initialize fully
+  test_connection ${test_port}
+
+  info "Testing whether \$HOME points to web server Document root"
+  homef=$(docker exec -it $(cat ${cid_file}) /bin/bash -c "echo \$HOME")
```

Comment:
please use at the beginning `local homef` so that variable is not expanded into whole script.

Otherwise LGTM.

Reply:
Changed